aws/util.sh

```diff
@@ -5,7 +5,40 @@
 # All providers are expected to support the following functions:
 #	util::get_instance_info
 #	util::copy_file
-#
+
+# Helpers #
+# These functions implement lower leverl operations required for deploying
+# and destroying Amazon Web Service EC2 instances.  These functions should be
+# considered private, scoped to the provider.
+
+function __run_instance() {
+	# TODO parameterize options values
+	aws ec2 run-instances \
+	--count 2 \
+	--image-id ami-2fb42b39 \
```

Comment:
should the image-id be a const (readonly var) or can it change?

Reply:
Next iteration, these values will be stored in a config and over-rideable via option and env var.  What do you think about breaking configs into gce, aws and common files?